test/e2e/specs/i18n/i18n__editor.ts

```diff
@@ -265,50 +266,33 @@ describe( 'I18N: Editor', function () {
 			}
 		} );
 
-		const testAccount = new TestAccount( accountName );
 		await testAccount.authenticate( page );
+		restAPIClient = new RestAPIClient( testAccount.credentials );
 
 		editorPage = new EditorPage( page, { target: features.siteType } );
 	} );
 
 	describe.each( locales )( `Locale: %s`, function ( locale ) {
-		describe( 'Set up', function () {
-			it( `Change UI language to ${ locale }`, async function () {
-				await Promise.all( [
-					page.waitForNavigation( { url: '**/home/**', waitUntil: 'load' } ),
-					page.goto( DataHelper.getCalypsoURL( '/' ) ),
-				] );
-
-				const changeUILanguageFlow = new ChangeUILanguageFlow( page );
-				await changeUILanguageFlow.changeUILanguage( locale as LanguageSlug );
-
-				await Promise.all( [
-					page.waitForNavigation( { url: '**/home/**', waitUntil: 'load' } ),
-					page.goto( DataHelper.getCalypsoURL( '/' ) ),
-				] );
-			} );
+		beforeAll( async function () {
+			await restAPIClient.setMySettings( { language: locale } );
+			await page.reload( { waitUntil: 'networkidle', timeout: 20 * 1000 } );
+		} );
```

Comment:
@karenroldan The solution to the issue encountered in `ChangeUILanguageFlow` is to use the API to set the language.

We use the API throughout the framework already and despite it not being entirely representative of user interaction, I believe it's acceptable here because we're not testing the language-changing flow but rather that strings on the page are updated as expected.

Let me know if you agree with this solution!

Reply:
> I believe it's acceptable here because we're not testing the language-changing flow but rather that strings on the page are updated as expected.

I agree with this. It's an efficient way to ensure that translations are working properly. Thanks for working on this!

Reply:
> I believe it's acceptable here because we're not testing the language-changing flow but rather that strings on the page are updated as expected.

I agree with this. It's an efficient way to ensure that translations are working properly. Thanks for working on this!

Reply:
At some point, I will extract this logic into a helper function as this same snippet is repeated elsewhere at least once.